bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         echo -n "Fixing audit permissions."
         chmod 0700 /var/log/kube-apiserver
         echo -n "Waiting for port :6443 and :6080 to be released."
-        while [ -n "$(lsof -ni :6443)$(lsof -ni :6080)" ]; do
+        while [ -n "$(ss -Htan '( sport = 6443 or sport = 6080 )')" ]; do
```

Comment:
can you also update the bootstrap container? I think we have it there as well (some sibling directory).

Reply:
the bootstrap doesn't have any port waiting - should I add it in?

Reply:
Let me check. I thought we do have it there.

Reply:
We don't. So nvmd. No need for bootstrapping then.